sparse/unit_test/Test_Sparse_TestUtils_RandCsMat.hpp

```diff
@@ -46,60 +46,60 @@
 
 namespace Test {
 template <class ScalarType, class LayoutType, class ExeSpaceType>
-void doCscMat(size_t m, size_t n, ScalarType min_val, ScalarType max_val) {
+void doCsMat(size_t m, size_t n, ScalarType min_val, ScalarType max_val) {
   auto expected_min    = ScalarType(1.0);
   int64_t expected_nnz = 0;
-  RandCscMat<ScalarType, LayoutType, ExeSpaceType> cm(m, n, min_val, max_val);
+  RandCsMatrix<ScalarType, LayoutType, ExeSpaceType> cm(m, n, min_val, max_val);
 
   for (int64_t i = 0; i < cm.get_nnz(); ++i)
     ASSERT_GE(cm(i), expected_min) << cm.info;
 
-  for (int64_t j = 0; j < cm.get_n(); ++j) {
-    for (int64_t i = 0; i < cm.get_col_len(j); ++i)
-      ASSERT_FLOAT_EQ(cm(cm.get_col_start(j) + i), cm(expected_nnz + i))
+  for (int64_t j = 0; j < cm.get_nelem(); ++j) {
```

Comment:
The element terminology from previous review is still there. Elements are associated with mesh.

Reply:
I am ok with this since it's in the unit tests.

Reply:
Please rename. We have to maintain unit tests too. Otherwise we will have to remember for the time this test lives it has the weird convention of using elements and vectors when it means row and columns or columns and rows. If you don't know, call it primary dimension and secondary one.

Reply:
Ok, thanks for clarifying.